### DISCUSSION TITLE
App Branding

### DISCUSSION THREAD

**Discussion by Natashka333** [timmyichen/dev-directory#45] [Ideas]
:brain: Let's brainstorm a **name**
:framed_picture: Any thoughts on **logo**
:door: and **domain name**

**Comment by timmyichen** [comment]
<img width="788" alt="image" src="https://user-images.githubusercontent.com/5304277/226149103-fc6a87a7-d393-4768-ad10-3ed50fb7c20f.png">
kek

**Reply by Caleb-Cohen** [reply]
DevConnect and CodeCrew are pretty good.

**Reply by DevinCLane** [reply]
I like CodeCrew

**Reply by windyScripts** [reply]
DevConnect sounds memorable.

**Comment by dev-skworks** [comment]
DevGather - keeps it in the same vein as Together(kinda, maybe)

DevCo - Co represents a bunch of words here such as Connection, Collaboration and Cooperation. Kinda fits the theme

**Reply by Caleb-Cohen** [reply]
I like DevGather

**Comment by wo1vin** [comment]
- Here to Help
- We Go Get 
- DevDocs
- Code Weenies 
- Dev Direct (might as well)
- Code Connect
- Programming Plug 🔌
- Fullstack Friends

**Reply by Caleb-Cohen** [reply]
I like Dev Direct and Code Connect

**Comment by Caleb-Cohen** [comment]
- CodeCompanions
- DevDirectory

Then more abstract like:

- Connect
- Commune
- Comingle

**Reply by DevinCLane** [reply]
Dev Directory sounds familiar, i like it

**Comment by windyScripts** [comment]
- devDuo/ devDual
- duoLogic
- develUp
- coNet

**Comment by Natashka333** [comment]
- DevMatch
- DevFinder
- DevPoint
- DevSpot
- DevSquad
- Directify
- PairUp

**Reply by Caleb-Cohen** [reply]
I like DevMatch, DevFinder, and PairUp

**Reply by DevinCLane** [reply]
+1 PairUp

**Reply by windyScripts** [reply]
PairUp sounds memorable, +1

**Reply by MainlyColors** [reply]
+1 PairUp

**Comment by MainlyColors** [comment]
I asked chat GPT, I think "DevBuddy" is interesting among this list 

![image](https://user-images.githubusercontent.com/68509315/226200738-e04fa83b-c5a1-4508-9d75-b25d25076f48.png)

**Comment by timmyichen** [comment]
i think anything *connect (like devconnect) makes us required to host a conference for it so we can [introduce ourselves...](https://www.youtube.com/watch?v=yIL9wLxG01M&t=14s)

**Comment by Natashka333** [comment]
Here is the link for the name ranking poll who haven't voted yet 
https://rankit.vote/vote/pBVludgSlfVduQjyYRkU

**Comment by Natashka333** [comment]
looks like we have the name winner: **PairUp**:fire:
click the link to see poll results  :nice: https://rankit.vote/vote/pBVludgSlfVduQjyYRkU

**Comment by Natashka333** [comment]
Here are some first and fresh logo ideas
![Screen Shot 2023-03-29 at 2 10 50 PM](https://user-images.githubusercontent.com/102708144/228953139-469b7b72-e4de-4b22-9425-393c0ee9d2ff.png)
![Screen Shot 2023-03-29 at 2 11 21 PM](https://user-images.githubusercontent.com/102708144/228953151-c014a882-bb03-4f5c-a305-b4f4146747b7.png)
![Screen Shot 2023-03-29 at 2 11 00 PM](https://user-images.githubusercontent.com/102708144/228953163-7508a7fd-e370-4e41-880e-348511a30e74.png)

**Reply by Caleb-Cohen** [reply]
I like bunnies because it's fun and they are coming up out of the rabbit hole.

**Reply by DevinCLane** [reply]
I like the bunnies!

**Reply by jmoldyvan** [reply]
I also like the bunnies... could they be holding hands maybe?

**Reply by DevinCLane** [reply]
I also think the two developers with a circle is the most logical but the bunnies are cuter
